modules/bootstrap/variables.tf

```
@@ -42,6 +42,12 @@ variable "files" {
   type        = map(string)
 }
 
+variable "files_md5" {
+  description = "Optional map of MD5 hashes. Normally the map should be empty, it is intended to be used for the files created in the same Terraform run as this module, which would not be automatically hashed. The keys of the map should be same or all of the keys of `files` input, while the values should be MD5 hashes of file contents. For example `{\"dir/my.txt\" = \"6f7ce3191b50a58cc13e751a8f7ae3fd\"}`"
```

```suggestion
  description = "Optional map of MD5 hashes. Normally the map should be empty, it is intended to be used for the files created in the same Terraform run as this module, which would not be automatically hashed. The keys of the map should be the same as the keys of the `files` input, while the values should be MD5 hashes generated from file contents. For example `{\"dir/my.txt\" = \"6f7ce3191b50a58cc13e751a8f7ae3fd\"}`"
```

---

Uh, I've meant something else. Let me know how would the text below work for you?

```suggestion
  description = "Optional map of MD5 hashes of file contents. Normally the map could be all empty, because all the files that exist before the `terraform apply` will have their hashes auto-calculated. This input is necessary only for the selected files which are created/modified within the same Terraform run as this module. The keys of the map should be identical with selected keys of the `files` input, while the values should be MD5 hashes of the contents of that file. For example `{\"dir/my.txt\" = \"6f7ce3191b50a58cc13e751a8f7ae3fd\"}`"
```

---

Indeed, all clear now, thank you 👍